index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style1.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,500&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="nav-container">
+        <div class="wrapper">
+            <img class="headerimgHolder" src="assets/image_geometry_1.png" alt="headerimgHolder" />
+            <div class="ic_menu">
+                <a><img src="assets/ic_menu.png" alt="menu icon"></a>
+            </div>
+        </div>
+
+    </div>
```

Comment:
- [ ] Please make sure the container of the section is a header tag as stated in the requirement👍